plugins/woocommerce/src/Blocks/BlockTypes/AddToCartWithOptions/AddToCartWithOptions.php

```diff
@@ -202,6 +202,24 @@ protected function render( $attributes, $content, $block ) {
 				)
 			);
 
+			wp_interactivity_state(
+				'woocommerce',
+				array(
+					// Use camelCase for error messages generated from the frontend,
+					// and snake_case for error messages generated from the backend.
```

Comment:
I'm not 100% sure about this, but using snake_case for errors generated in the frontend might conflict with errors from the server. At the same time, I reckon it's a bit weird mixing camelCase and snake_case on the same object.

An alternative would be having two `errorMessages` arrays, one for client-side errors and another one for server-side errors. But considering we only have two errors right now, it felt like an unnecessary optimization at this point. But happy to change my mind. :thinking:

Reply:
I don't have a strong opinion on it. We can start as-is and see how it evolves. Assuming the total number of errors won't be too big (dozen maybe?), then keeping this convention is not a problem. If it grows higher for any reason we can revisit.